modules/installation-configuration-parameters.adoc

```diff
@@ -1735,55 +1608,104 @@ ifdef::alibabacloud[]
+|String list.
+
+|`controlPlane.platform.alibabacloud.imageID`
+|The imageID used to create the ECS instance. ImageID must belong to the same region as the cluster.
+|String.
+
+|`controlPlane.platform.alibabacloud.instanceType`
+|InstanceType defines the ECS instance type. Example: `ecs.g6.xlarge`
+|String.
+
+|`controlPlane.platform.alibabacloud.systemDiskCategory`
+|Defines the category of the system disk. Examples: `cloud_efficiency`,`cloud_essd`
+|String.
+
+|`controlPlane.platform.alibabacloud.systemDisksize`
+|Defines the size of the system disk in gibibytes (GiB).
+|Integer.
+
+|`controlPlane.platform.alibabacloud.zones`
+|The list of availability zones that can be used. Examples: `cn-hangzhou-h`, `cn-hangzhou-j`
+|String list.
+
 |`platform.alibabacloud.region`
 |Required. The Alibaba Cloud region where the cluster will be created.
 |String.
 
 |`platform.alibabacloud.resourceGroupID`
-|The ID of an already existing resource group where the cluster will be installed. If empty, the installation program will create a new resource group for the cluster.
+|The ID of an already existing resource group where the cluster will be installed. If empty, the installer will create a new resource group for the cluster.
 |String.
 
 |`platform.alibabacloud.tags`
 |Additional keys and values to apply to all Alibaba Cloud resources created for the cluster.
 |Object.
 
 |`platform.alibabacloud.vpcID`
-|The ID of an already existing VPC where the cluster should be installed. If empty, the installation program will create a new VPC for the cluster.
+|The ID of an already existing VPC where the cluster should be installed. If empty, the installer will create a new VPC for the cluster.
 |String.
 
 |`platform.alibabacloud.vswitchIDs`
-|The ID list of already existing VSwitches where cluster resources will be created. The existing VSwitches can only be used when also using existing VPC. If empty, the installation program will create new VSwitches for the cluster.
+|The ID list of already existing VSwitches where cluster resources will be created. The existing VSwitches can only be used when also using an existing VPC. If empty, the installer will create new VSwitches for the cluster.
 |String list.
 
 |`platform.alibabacloud.defaultMachinePlatform.imageID`
-|For compute machines, the image ID that should be used to create ECS instance. If set, the image ID should belong to the same region as the cluster
+|For both compute machines and control plane machines, the image ID that should be used to create ECS instance. If set, the image ID should belong to the same region as the cluster.
 |String.
 
 |`platform.alibabacloud.defaultMachinePlatform.instanceType`
-|For compute machines, the configuration used when installing on Alibaba Cloud.
-|String. For example `ecs.g6.large`.
+|For both compute machines and control plane machines, the ECS instance type used to create the ECS instance. Example: `ecs.g6.large`
```

Comment:
Suggest to replace `ecs.g6.large` with `ecs.g6.xlarge` for `platform.alibabacloud.defaultMachinePlatform.instanceType`.

Reply:
@jianli-wei Done, thank you sir.